firefox-ios/Client/metrics.yaml

```diff
@@ -2712,6 +2712,25 @@ preferences:
       - fx-ios-data-stewards@mozilla.com
     expires: "2024-06-01"
 
+# Private Browsing
+private_browsing:
+  data_clearance_icon_tapped:
+    type: event
+    description: |
+      Records when the private browsing button is tapped.
+    extra_keys:
+      did_confirm:
+       type: string
```

Comment:
Could this be a boolean?

Reply:
That makes sense to me! Let me touch base with Andy on the requirements. Seems there's inconsistencies on whether we use boolean or string for these types in general. Do you have context in why?

String: https://dictionary.telemetry.mozilla.org/apps/firefox_ios/metrics/accessibility_voice_over
Boolean: https://dictionary.telemetry.mozilla.org/apps/firefox_ios/metrics/logins_sync_enabled

Reply:
@afurlan-firefox

Reply:
Andy confirmed that Boolean is preferred!~ thanks @dnarcese !